.github/workflows/main.yml

```diff
@@ -6,7 +6,7 @@ concurrency:
 on:
   workflow_dispatch:
   push:
-    branches: [ 'main' ]
+    branches: [ 'kk/add-lint-test' ]
```

Comment:
All mentions of `kk/add-lint-test` will be renamed to `main` once the PR is approved.